vaadin-testbench-core/src/main/java/com/vaadin/testbench/screenshot/ImageUtil.java

```diff
@@ -70,7 +70,8 @@ public static String encodeImageToBase64(BufferedImage image) {
      * Get luminance value for the given rgb value.
      * 
      * @param rgb
-     * @return
+     *            rgb value
```

Comment:
I get the feeling that we might want to explain that the red, green and blue byte values should be packed into an integer at given offsets somewhere here..

Reply:
something like `The RGB value is assumed to be taking up the low 24 bits of the integer sample, with red shifted by 16 bits, green by 8 bits and blue by 0 bits, ORed together. This function will not return the correct values for other channel orders.`